### PR TITLE
Update Passenger Showcase Guide

### PR DESCRIPTION
**Pull request by kvashchuka**
Solves #628

### CODE REVIEW

File: content/pages/guides/passenger-showcase-guide.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ If you have questions or need any help, come board the [Coding Train Discord](ht
 
 If you choose not to use the form above, you can also submit a Pull Request on GitHub. The first step is to associate your submission with a specific video. Don't worry too much if you aren't sure, just pick wherever you think it fits best! If your work relates to multiple videos just pick one!
 
-On the [GitHub repo homepage](https://github.com/CodingTrain/thecodingtrain.com), click the button in the upper left-hand corner where it says `main`. Type in the name of a branch (you can make this up!) and create the branch! In the example below I'm calling it `dan-showcase-submission`.
-
+First, you will need to [fork the repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository). On the [GitHub repo homepage](https://github.com/CodingTrain/thecodingtrain.com), click the button `Fork` in the upper-right corner. You will be prompted to the `Create a new fork` window, press `Create fork` in the bottom of the page. Now, you will have a fork of the original repo. Second, create a branch in your fork by clicking a button in the upper left-hand corner where it says `main`.
```
Comment (alin256) [inline content/pages/guides/passenger-showcase-guide.md:19]: Please split in two paragraphs for people who do not read :)

**Review by alin256 — APPROVED** [review]
Thanks. This seems to address the raised issue.

**Comment by alin256** [comment]
All checks and preview seem to be in-order. 
@shiffman or @kfahn22, can you merge this pull?

**Comment by shiffman** [comment]
This is fantastic, thank you @kvashchuka and @alin256 for reviewing!

**Comment by shiffman** [comment]
@allcontributors  add @kvashchuka for documentation

**Pull request merged**